h2o-core/src/main/java/water/parser/ParseSetup.java

```diff
@@ -27,6 +28,8 @@
   public static final int HAS_HEADER = 1;
   public static final int GUESS_COL_CNT = -1;
 
+  public BytesStats bytesStats;
```

Comment:
I didn't find BytesStats in master. What value does it have for the other parsers?